docs/project-setup/existing-projects.md

```diff
@@ -1,3 +1,55 @@
 # Existing Projects
 
-To Be Done
\ No newline at end of file
+To add Outrigger support to an existing project, you'll need to understand how to create a Docker Compose and an 
+`.outrigger.yml` file, how to configure DNS for your project's containers and how to configure volumes and persistent 
+storage if your project requires.
+
+## Docker Compose file(s)
+
+Outrigger is intended to work with either raw Docker commands or Docker Compose files, and we find things are best when
+orchestrating your local environments with Docker Compse files
+
+Be sure to read the [Docker Compose compose file documentation](https://docs.docker.com/compose/compose-file/)
+so that you have a handle on how to setup the various configuration.
+
+Please see the [Outrigger Examples Repository](https://github.com/phase2/outrigger-examples) for how to setup various 
+technologies within Outrigger.  Specifically, the [Drupal 8 example](https://github.com/phase2/outrigger-examples/tree/master/drupal8) 
+covers all of the high points.
+
+### Project Services
+
+The essential point is that you will need to define a Docker Compose Service for each of the components in your architecture.
+Web/API Server, database, cache/key-value store, search server, proxy cache, mail server, etc. Each of those will be configured
+as a service in your `docker-compose.yml` file.
+
+### Build Container
+
+The default build image for Outrigger is `outrigger/build`. It contains a lot of the tools required for modern web development, 
```

Comment:
Could use a lead in sentence here as someone might not yet know why they would want a build container.

Maybe something like: "In addition to containers for all of your project services, we find it useful to have a container we refer to as the build container. This container supports all of the needs of development for your project. We often configure the build container ........ Check out an example of a build.yml Docker Compose file which uses the outrigger/build image for the build container. Our outrigger/build image contains a lot of the tools required for modern web development, things like ....."